Uplift/Models/Equipment.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  Equipment.swift
+//  Uplift
+//
+//  Created by Belle Hu on 2/25/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import UpliftAPI
+
+/// Model representing a facility's equipment.
+struct Equipment: Hashable {
+
+    // MARK: - Properties
+
+    /// Accessibility for this equipment.
+    let accessibility: AccessibilityType?
+
+    /// The type of this equipment.
+    let equipmentType: EquipmentType
```

Comment:
```suggestion
    let equipmentType: EquipmentType?
```
Let's make this an optional instead of hard coding a default type.